### ISSUE TITLE
[ENG-222] libavresample-dev fails to install

### ISSUE THREAD

**Issue by Mamun-Developer** [spacedriveapp/spacedrive#369]
### Describe the bug

When we execute `./.github/scripts/setup-system.sh` the follwing error comes up

> Package libavresample-dev is not available, but is referred to by another package.
> This may mean that the package is missing, has been obsoleted, or
> is only available from another source
> 
> E: Package 'libavresample-dev' has no installation candidate
> An error occurred while performing the task on line 78
> Setup for Spacedrive development failed

### Reproduction

1. Execute `./.github/scripts/setup-system.sh`

### Expected behavior

Must be successful installation.

### Platform and versions

```Shell
7.9.5
cargo 1.63.0 (fd9c4297c 2022-07-01)
rustc 1.63.0 (4b91a6ea7 2022-08-08)

OS: Ubuntu 22.04
Processor: AMD® Ryzen 7 5800h with radeon graphics × 16
Memory: 16GB
```

### Stack trace

```Shell
mamun@mamun-A35S:~/spacedrive$ ./.github/scripts/setup-system.sh
Setting up your system for Spacedrive development!
Detected 'apt' based distro!
Hit:1 https://apt.releases.hashicorp.com jammy InRelease
Hit:2 https://download.docker.com/linux/ubuntu jammy InRelease                                                                                                                       
Hit:3 https://dl.google.com/linux/chrome/deb stable InRelease                                                                                                                        
Hit:4 http://packages.microsoft.com/repos/code stable InRelease                                                                                                                      
Hit:5 http://security.ubuntu.com/ubuntu jammy-security InRelease                                                                                                                     
Hit:6 https://deb.nodesource.com/node_16.x jammy InRelease                                                                                                               
Hit:7 http://bd.archive.ubuntu.com/ubuntu jammy InRelease                                                                              
Hit:8 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease                                    
Hit:9 http://bd.archive.ubuntu.com/ubuntu jammy-updates InRelease                 
Hit:10 http://bd.archive.ubuntu.com/ubuntu jammy-backports InRelease              
Hit:11 https://ppa.launchpadcontent.net/git-core/ppa/ubuntu jammy InRelease       
Reading package lists... Done                              
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package libavresample-dev is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'libavresample-dev' has no installation candidate
An error occurred while performing the task on line 78
Setup for Spacedrive development failed
```

**Comment by Brendonovich** [comment]
@oscartbeaumont Would be great if you could check this out

**Comment by mmattbtw** [comment]
is this the same as #67?

**Comment by PyRo1121** [comment]
Looking over when I attempted on Ubuntu I got the same error and noticed it appears the libavresample-dev was removed as a dep after version 4.3.4. By updating and removing that dep for apt based OS it build and runs fine with no issues now.

**Comment by PyRo1121** [comment]
This issue was resolved with [PR #416](https://github.com/spacedriveapp/spacedrive/pull/416) and I believe can be closed at this point.
